service/controller/internal/vmsscheck/vmss.go

```diff
@@ -13,6 +13,10 @@ import (
 )
 
 const (
+	// Provisioning States.
+	provisioningStateFailed    = "Failed"
+	provisioningStateSucceeded = "Succeeded"
```

Comment:
Where do these come from? Are they used?

Reply:
they are used, they were set in a file that was deleted